### ISSUE TITLE
[cli] build err and solution

### ISSUE THREAD

**Issue by huangwei2013** [rancher/rancher#24552]
when buiding, got the errors below：
```
[root@iZm5efctez2mq4wk8wbhsyZ cli]# go build main.go 
# github.com/rancher/cli/cmd
cmd/multiclusterapp.go:723:5: app.Wait undefined (type *"github.com/rancher/types/client/management/v3".MultiClusterApp has no field or method Wait)
cmd/multiclusterapp.go:724:5: app.Timeout undefined (type *"github.com/rancher/types/client/management/v3".MultiClusterApp has no field or method Timeout)
```

and if  **commented out** the 2 lines，build & run will fine.

Here is server infos：

```
rancher/cli:master
go version go1.13.4 linux/amd64
centos 7.6(Linux iZm5efctez2mq4wk8wbhsyZ 3.10.0-862.14.4.el7.x86_64 rancher/rancher#1 SMP Wed Sep 26 15:12:11 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux)
```

**Comment by dramich** [comment]
This is due to the dependencies in go.mod not lining up with what is in the vendor dir. If you run `go build --mod vendor` it will work until my fix is merged.

**Comment by dramich** [comment]
available in v2.4.0-rc2

**Comment by jiaqiluo** [comment]
Validated in rancher/cli https://github.com/rancher/cli, master branch 859a86e9148d7ef611e5814ff5682f8dd1024d12

Run `go build main.go` and do not see the mentioned errors anymore, and the binary is built successfully.
